projects/fsastorefrontlib/src/cms-components/sync-pilot/sync-pilot-connection.component.ts

```diff
@@ -35,6 +35,7 @@ export class SyncPilotConnectionComponent implements OnInit, OnDestroy {
 
   protected subscription = new Subscription();
   protected readonly ownerId = 1;
+  protected readonly groupId = 1;
```

Comment:
Let's finally move these 2 properties to some kind of sync pilot config file.
To have one centralized place where someone can modify owner and group id